featuretools/entityset/entityset.py

```diff
@@ -1151,10 +1151,27 @@ def add_last_time_indexes(self, updated_dataframes=None):
             self.dataframe_dict[df.ww.name] = df
 
         self.reset_data_description()
+
     # ###########################################################################
-    # #  Other ###############################################
+    # #  Pickling ###############################################
     # ###########################################################################
+    def __getstate__(self):
+        return {
+            **self.__dict__,
+            'schemas': {df_name: df.ww.schema for df_name, df in self.dataframe_dict.items()}
+        }
+
+    def __setstate__(self, state):
+        schemas = state.pop('schemas')
+        for df_name, df in state.get('dataframe_dict', {}).items():
+            if schemas[df_name] is not None:
+                df.ww.init(schema=schemas[df_name])
```

Comment:
I *think* we can run init here with `validate=False` since we should know the schema is valid for the dataframe, and this could save a little time.
```
df.ww.init(schema=schemas[df_name], validate=False)
```